README.md

```diff
@@ -65,6 +67,15 @@ s.receiving.instructions_recipients(id=1234) # Get email recipients and instruct
 s.receiving.items(id=1234) # Get the contents of this receiving
 s.receiving.shipments(id=1234) # Get shipping dimension and container information
 s.receiving.trackings(id=1234) # Get tracking information about this receiving.
+s.receiving.labels(id=1234) # Get labels about a given receiving.
+s.returns.list() # Get a list of returns.
+s.returns.create(json=your_json) # create a new return or multiple returns
+s.returns.get(id=1234) # returns a return resource for the given id.
+s.returns.cancel(id=1234) # cancel a return.
+s.returns.holds(id=1234) # returns a list of holds for a given receiving.
```

Comment:
@clarkf i think the plural naming convention is a valid option here since we can't go with the singular. Minor typo in this documentation line, "receiving" instead of "return"